tests/test_aggregate_by_key.py

```diff
@@ -3784,24 +3784,31 @@ def test_aggregation_unique_fields():
         ).run()
 
 
-def test_fixed_window_aggregation_with_first_and_last_aggregates():
+@pytest.mark.parametrize("timestamp", [True, False])
```

Comment:
i'm not following, where is the timestamp parameter being used?

Reply:
One line below.